Service/ZgwToVrijbrpService.php

```diff
@@ -359,24 +437,26 @@ public function zgwToVrijbrpHandler(array $data, array $configuration): array
      *
      * @return array The response body of the outgoing call, or an empty array on error.
      */
-    private function synchronizeTemp(Synchronization $synchronization, array $objectArray): array
+    public function synchronizeTemp(Synchronization $synchronization, array $objectArray, string $location): array
     {
         $xmlEncoder = new XmlEncoder(['xml_root_node_name' => 'soapenv:Envelope']);
         $objectString = $xmlEncoder->encode($objectArray, 'xml', ['xml_encoding' => 'utf-8', 'remove_empty_tags' => true]);
 
+
         $this->logger->info('Sending message with body '.$objectString);
+
         try {
             $result = $this->callService->call(
```

Comment:
If we use the $source->location often, this should be the default for callService->call.
And if it already is why do we still do ->synchronizeTemp(sync, array, source->getLocation()) and have string $location for this function?

Reply:
The location variable passed to callservice only contains the endpoint on the source, the base location of the source is extracted from the source by the callservice

Reply:
Oke, but i still only see it being used like this:
synchronizeTemp($synchronization, $objectArray, $source->getLocation())
So if we give $source to the callservice->call function, we should not have to do this^ because callservice has a source and can do it?

Reply:
Or are there cases right now where we want to use something else than $source->getLocation() ?